docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -1181,9 +1183,9 @@ paths:
           description: |
             Will query all subscriptions that the given user or service can read from, based on authorization section. Service or user
             should be listed explicitly on authorization section.
+            example: "service:stups_nakadi-ui"
```

Comment:
why did you move example into description ?

Reply:
Not supported by Swagger 2.0.
We are using OpenAPI 3.0 syntax without actually adopting it completely.

Reply:
can we maybe make it from the capital letter and add a new line that it will be separated from the description ?